django/api/email.py

```diff
@@ -11,3 +12,30 @@ def config():
         "SENDER_NAME": "CleanBC Go Electric",
         "SEND_EMAIL": os.getenv("SEND_EMAIL", "False") == "True",
     }
+
+
+class FOUR_THOUSAND_REBATE(Enum):
```

Comment:
Lets move these to a constants.py file and add some comments to explain where these values come from.